utils/nctl/sh/assets/compile_client.sh

```diff
@@ -33,6 +33,13 @@ popd || exit
 pushd "$NCTL_CASPER_CLIENT_HOME" || \
     { echo "Could not find the casper-client-rs repo - have you cloned it into your working directory?"; exit; }
 
+while getopts 'd' opt; do 
+    case $opt in
+        d ) export NCTL_COMPILE_TARGET="debug";;
```

Comment:
Hmm, just thinking out loud. Maybe we should consider being consistent with the approach taken by Rust ecosystem and by default compile in `debug` and support the `--release` command line option?

That might be a too much of a lift, though, since `NCTL` is already wired-up in may places that expect it to compile in release by default.

So treat it just as something for consideration.

CC: @TomVasile